BtConnectorLib/btconnectorlib2/src/main/java/org/thaliproject/p2p/btconnectorlib/WifiServiceSearcher.java

```diff
@@ -11,13 +11,12 @@
 import android.os.CountDownTimer;
 import android.os.Handler;
 import android.util.Log;
-
 import org.json.JSONObject;
-
-import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Iterator;
 import java.util.List;
 import java.util.Random;
+import java.util.concurrent.CopyOnWriteArrayList;
 
 import static android.net.wifi.p2p.WifiP2pManager.WIFI_P2P_PEERS_CHANGED_ACTION;
 
```

Comment:
Why do we have GetLastError and lastError? Nobody seems to ever use them including the Thali_CordovaPlugin library?